dirsrvtests/tests/suites/syncrepl_plugin/openldap_test.py

```diff
@@ -45,18 +46,25 @@ def test_syncrepl_openldap(topology):
         1. Success
     """
     st = topology.standalone
+    # Ensure entryuuid is setup
+    plug = EntryUUIDPlugin(st)
+    task = plug.fixup(DEFAULT_SUFFIX)
```

Comment:
Note that this testcase fails for me. The entryUUID tasks container does not exist. 

<pre>
[04/Dec/2020:10:09:14.190071590 +0100] conn=1 op=5 ADD dn="cn=entryuuid_fixup_2020-12-04T10:09:14.188531,cn=entryuuid task,cn=tasks,cn=config"
[04/Dec/2020:10:09:14.192727883 +0100] conn=1 op=5 RESULT err=32 tag=105 nentries=0 wtime=0.000286806 optime=0.002672704 etime=0.002953848

</pre>

Reply:
Have you set --enable-rust? EntryUUID only works if you have rust ... 

Saying that I should check that the tests in this suite *only* operate if enable_rust is true, so that's a good point.